docs/rfc/0008-ubersimplify-command.md

```
@@ -0,0 +1,205 @@
+  - location: <path>:<line>
+    severity: blocker | suggestion
+    lens: Reuse | Quality | Efficiency
+    summary: <1-line>
+    detail: <prose>
+```
+
+### 5.2 Per-chunk fixer aggregate (`chunk-NNN-fixer.md`)
+
+`aggregate.py` dedups the chunk's lens findings by `file:line` (merging across lenses: `lens: Reuse+Quality`, ` | `-joined summaries/details, severity = max), then wraps the result in the envelope `code-fixer` validates:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)